apps/oxlint/src/output_formatter/checkstyle.rs

```diff
@@ -17,6 +17,10 @@ impl InternalFormatter for CheckStyleOutputFormatter {
         writeln!(writer, "flag --rules with flag --format=checkstyle is not allowed").unwrap();
     }
 
+    fn lint_command_info(&self, _: &super::LintCommandInfo) -> Option<String> {
```

Comment:
You can make the trait default implementation return `None` so you don't need to add these blanket implementations.